app/config/security.yml

```diff
@@ -8,6 +8,23 @@ security:
     encoders:
         Sylius\Component\User\Model\UserInterface: sha512
     firewalls:
+        admin:
+            switch_user: true
+            context: user
+            pattern: /admin/.*
```

Comment:
just a quick thought from a security-through-obscurity perspective, should we consider not using such a common backend access url?